lib/gui/app.js

```diff
@@ -130,7 +129,7 @@ app.run((AnalyticsService, OSWindowProgressService, FlashStateModel) => {
   });
 });
 
-app.run(($timeout, DriveScannerService, DrivesModel, ErrorService, DriveSelectorService) => {
+app.run(($timeout, DriveScannerService, DrivesModel, ErrorService) => {
```

Comment:
Looks like we can remove the `require('./components/drive-selector/drive-selector'),` from the top dependencies as well.